docs/src/python-api.rst

```diff
@@ -0,0 +1,6 @@
+.. _python-api:
+
+Python API
+==========
+
+Not sure what goes here, all the files are underscored.
```

Comment:
Everything in there: https://github.com/Luthaf/vesin/blob/bb724f0d53425fcfe911511bef510a50ec6f58de/python/vesin/src/vesin/__init__.py#L5-L8